### PR TITLE
Ref[Tracker][53503][Create admin static pages]

### PR DESCRIPTION
**Pull request by congth2001**
## Related Tickets
- [#53503](https://edu-redmine.sun-asterisk.vn/issues/53503)

## WHAT (optional)
- Create admin static pages

## HOW
- Using namespace for admin
- Using HTML, CSS, Bootstrap to create static pages
## Evidence (Screenshot or Video)
Screenshot from 2022-09-20 14-14-02

### CODE REVIEW

**Comment by congth2001** [comment]
ready

File: app/assets/stylesheets/admin/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+a.download {
+  background: #fff;
+  color: #7386D5;
+}
+
+a.article, a.article:hover {
+  background: #6d7fcc !important;
+  color: #fff !important;
```
Comment (ghost) [inline app/assets/stylesheets/admin/static_pages.scss]: bỏ important đc ko em, đặt biến màu

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -10,8 +10,13 @@
...
  *= require_self
  *=
  */
  @import "font-awesome";
+ @import "bookings";
+ @import "color";
+ @import "sessions";
+ @import "custom";
+ @import "shared";
+ @import "static_pages";
```
Comment (ghost) [inline app/assets/stylesheets/application.scss]: Dùng '

File: app/views/layouts/admin/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" id="sidebarCollapse" class="btn btn-info navbar-btn">
+        <i class="glyphicon glyphicon-align-left"></i>
+        <span><%= t(".toggle") %></span>
+      </button>
+      <%= link_to "EN", locale:"en" %>
+                <span>|</span>
+                <%= link_to "VN", class: "active", locale:"vi" %>
```
Comment (ghost) [inline app/views/layouts/admin/_header.html.erb]: sai indent

File: app/views/layouts/admin/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" id="sidebarCollapse" class="btn btn-info navbar-btn">
+        <i class="glyphicon glyphicon-align-left"></i>
+        <span><%= t(".toggle") %></span>
```
Comment (ghost) [inline app/views/layouts/admin/_header.html.erb]: bỏ ()

**Review by ghost — COMMENTED** [review]
cmt

**Comment by congth2001** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**